src-cpp/rdkafkacpp_int.h

```diff
@@ -498,11 +554,22 @@ class ConfImpl : public Conf {
       return Conf::CONF_OK;
     }
 
+  Conf::ConfResult get(CertVerifyCb *&ssl_cert_verify_cb) const {
+      if (!rk_conf_)
+      return Conf::CONF_INVALID;
```

Comment:
fix indent

Reply:
This is indented like the other ones above it.  Do you really want me to make this on different?